### ISSUE TITLE
Lodash error

### ISSUE THREAD

**Issue by ararog** [kimxogus/react-native-version-check#2]
I'm doing the following

```
    VersionCheck.needUpdate()
      .then(res => {
        if (res.isNeeded) {
          Alert.alert(
            'You must upgrade',
            'There is an updated version available.',
            [
              {
                text: 'Upgrade',
                onPress: () => {
                  Linking.openURL(VersionCheck.getStoreUrl())
                }
              }
            ]
          )
        }
      })
```

But I'm getting the following error:

( is not a function. (In '(0,_lodash5.defaultsDeep)', '(' is undefined)

Did you know any workaround to fix this?

**Comment by kimxogus** [comment]
I guess that updated dependencies are not applied.
Did you try `react-native start -- --reset-cache`?

**Comment by ararog** [comment]
Just tried that, no luck.

**Comment by kimxogus** [comment]
I'm so sorry for that issue. I guess that there was a problem with my tests.
I've just found and fixed this issue and published `v1.1.2` which works well.
Please check with `v1.1.2`.

**Comment by ararog** [comment]
Worked like a charm! Thanks!
